scm/src/GFS_typedefs.F90

```diff
@@ -2,34 +2,33 @@
 
```

Comment:
Many whitespace changes were included to sync up with GFS_typedefs.F90 from FV3 to make looking at future changes easier.